packages/bpk-component-rating/src/BpkRating.scss

```
@@ -21,20 +21,38 @@
 .bpk-rating {
   display: flex;
 
+  &--component {
```

```suggestion
  &__component {
```
In BEM `--` is for modifiers, this should be an element so `__`.

---

Done